packages/apps/dev-wallet/src/App/Layout/SideBar.tsx

```diff
@@ -0,0 +1,159 @@
+import {
+  MonoContacts,
+  MonoContrast,
+  MonoDashboardCustomize,
+  MonoDataThresholding,
+  MonoLogout,
+  MonoNetworkCheck,
+  MonoSwapHoriz,
+  MonoTableRows,
+  MonoWallet,
+} from '@kadena/kode-icons/system';
+
+import { NetworkSelector } from '@/Components/NetworkSelector/NetworkSelector';
+import { useWallet } from '@/modules/wallet/wallet.hook';
+import {
+  Button,
+  ContextMenu,
+  ContextMenuItem,
+  Themes,
+  useTheme,
+} from '@kadena/kode-ui';
+import {
+  SideBarItem,
+  SideBarItemsInline,
+  SideBarTree,
+  SideBarTreeItem,
+  SideBar as SideBarUI,
+  useSideBar,
+} from '@kadena/kode-ui/patterns';
+import { FC } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+export const SideBar: FC = () => {
+  const { theme, setTheme } = useTheme();
+  const { isExpanded } = useSideBar();
+  const navigate = useNavigate();
+  const { lockProfile } = useWallet();
+
+  const toggleTheme = (): void => {
+    const newTheme = theme === Themes.dark ? Themes.light : Themes.dark;
+    setTheme(newTheme);
+  };
+
+  const handleLogout = () => {
+    lockProfile();
+  };
+  return (
+    <SideBarUI
+      appContext={
+        <SideBarItem
+          visual={<MonoNetworkCheck />}
+          label="Select network"
+          onPress={() => {}}
```

Comment:
Are we going to add another function later?

Reply:
![image](https://github.com/user-attachments/assets/157cc029-fe9a-4673-b51f-a7263707e6ec)

This is indeed a dynamic area where we will have different callbacks later on